src/web/templates/member/screen-o-death.js

```diff
@@ -1,6 +1,17 @@
 <script type="text/javascript">
   $(document).ready(function() {
    $('a[data-toggle="tab"]:first').trigger("shown.bs.tab");
-
+   $('.btn-delete-monitor').click(function(e){
+       var monitorName = $(e.target).attr('data');
+       if (!window.confirm('Are you sure you want to delete monitor '+ monitorName + '?')) {
```

Comment:
Any particular reasons?
You'd essentially be recreating the behavior of the javascript Confirm dialog, which would not be blocking and would require extra work to make it accessible and keyboard friendly. Also, more code to maintain.